Sources/Danger/BitBucketCloud.swift

```diff
@@ -204,7 +204,7 @@ extension BitBucketCloud {
             case user
             case updatedOn = "updated_on"
             case type
-            case id // swiftlint:disable:this identifier_name
+            case id 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 207;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)